src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsExecution.java

```diff
@@ -108,14 +107,14 @@ public boolean start() throws Exception {
 
     /**
      * TODO JAVADOC
+     *
      * @param workspaceId
      * @throws IOException
      */
     private void registerFingerprint(String workspaceId) throws IOException {
-        FingerprintMap map = Jenkins.getActiveInstance().getFingerprintMap();
-        Fingerprint f = map.get(workspaceId);
+        Fingerprint f = Jenkins.getActiveInstance()._getFingerprint(workspaceId);
         if (f == null) {
-            return;
+            throw new AbortException("Couldn't find any Fingerprint for: " + workspaceId);
```

Comment:
I'm not very sure how I should handle this particular case.

If this exception occurs (although it may be an edge case scenario), when the user will try to browse the workspace for this particular fingerprint id (e.g. JENKINS_ROOT/exws/browse/wrongFingerPrintId/), he will get a nice error message

<img width="768" alt="screen shot 2016-09-01 at 9 56 45 pm" src="https://cloud.githubusercontent.com/assets/5720977/18180313/01ef505e-708f-11e6-99fe-51a79cfc81b2.png">